docs/abbr.md

```diff
@@ -33,10 +33,11 @@ Note that there are always exceptions, especially when we try to comply with the
 |                  | Convert between concepts            | 2              | name2idx(), label2idx(), seq2seq                 |
 | **Aggressive**   |                                     |                |                                                  |
 |                  | function                            | f              |                                                  |
-|                  | input                               | x              |                                                  |
+|                  | torch input                         | x              |                                                  |
 |                  | key                                 | k              |                                                  |
 |                  | value                               | v              |                                                  |
 |                  | index                               | i              |                                                  |
+|                  | lambda var                          | o              |                                                  |
```

Comment:
These two uses of `o` could be combined, e.g. "generic object argument e.g in list comprehensions and lambdas"

Reply:
done